oldzip_test.go

```diff
@@ -3,6 +3,7 @@ package xiter
 import "testing"
 
 func BenchmarkOldZip(b *testing.B) {
+	b.ReportAllocs()
```

Comment:
Is there any particular reason to use this instead of just passing `-benchmem` to `go test`?